qtpy/__init__.py

```diff
@@ -52,6 +52,9 @@
 
 import os
 
+# Version of QtPy
+__version__ = "1.1.dev0"
```

Comment:
`__version__` is defined here:

https://github.com/spyder-ide/qtpy/blob/master/qtpy/_version.py#L2

Please use that here instead :-)